custom_platforms.js

```diff
@@ -63,25 +516,16 @@ module.exports = {
         "path": "extension/chromium",
         "configuration": {
             "removeRulePatterns": [
-                // Do not exclude scriptlets which contain '$$' when excluding '$$' and '$@$' rules
-                // https://github.com/AdguardTeam/FiltersRegistry/issues/731
-                "^((?!#%#).)*\\$\\$|\\$\\@\\$",
-                "\\$(.*,)?replace=",
-                "important,replace=",
-                "\\$(.*,)?app",
-                "\\$network",
-                "\\$protobuf",
-                "important,protobuf",
-                "\\$extension",
-                ",extension",
-                "\\$hls=",
-                ",hls=",
-                "\\$jsonprune=",
-                ",jsonprune=",
-                "\\$referrerpolicy",
-                ",referrerpolicy",
-                "\\$content",
-                ",content"
+                ...HTML_FILTERING_MODIFIER_PATTERNS,
+                ...REPLACE_MODIFIER_PATTERNS,
+                ...APP_MODIFIER_PATTERNS,
+                ...NETWORK_MODIFIER_PATTERNS,
+                ...PROTOBUF_MODIFIER_PATTERNS,
+                ...EXTENSION_MODIFIER_PATTERNS,
+                ...HLS_MODIFIER_PATTERNS,
+                ...JSONPRUNE_MODIFIER_PATTERNS,
+                ...REFERRERPOLICY_MODIFIER_PATTERNS,
+                ...CONTENT_MODIFIER_PATTERNS,
```

Comment:
list of removeRulePatterns for `EXTENSION_CHROMIUM`, `EXTENSION_EDGE` and `EXTENSION_OPERA` are same. lets use a single constant for this list
```
const CHROMIUM_BASED_EXTENSION_PATTERNS = [ ... ];
```

Reply:
Done.